README.md

```diff
@@ -33,118 +34,583 @@ Without Sanctuary, one might have written:
 ```javascript
-S.tail([0, 1, 2, 3, 4]) // => Just([1, 2, 3, 4])
-S.tail([]) // => Nothing()
+> S.find(function(n) { return n < 0; }, [1, -2, 3, -4, 5])
+Just(-2)
+> S.find(function(n) { return n < 0; }, [1, 2, 3, 4, 5])
+Nothing()
 ```
 
-`init :: [a] -> Maybe [a]`
+<h4 name="pluck"><code><a href="https://github.com/plaid/sanctuary/blob/v0.4.0/index.js#L829">pluck :: String -> [{String: *}] -> [Maybe *]</a></code></h4>
+
+Takes a list of objects and plucks the value of the specified key
+for each object in the list. Returns the value wrapped in a Just
+if an object has the key and a Nothing if it does not.
 
-Returns a `Just` containing a list with all but the last element in the original list or a `Nothing` that list is empty.
 ```javascript
-S.init([0, 1, 2, 3, 4]) // => Just([0, 1, 2, 3])
-S.init([]) // => Nothing()
+> S.pluck('a', [{a: 1, b: 2}, {a: 4, b: 5}, {b: 3, c: 7}])
+[Just(1), Just(4), Nothing()]
 ```
 
-`find :: (a -> Boolean) -> [a] -> Maybe a`
+### Object
+
+<h4 name="get"><code><a href="https://github.com/plaid/sanctuary/blob/v0.4.0/index.js#L845">get :: String -> Object -> Maybe *</a></code></h4>
+
+Takes a property name and an object and returns Just the value of
+the specified property of the object if the object has such an own
+property; Nothing otherwise.
 
-Return a `Just` containing the first element in the list that satisfies the supplied predicate function or a `Nothing` if the item is not found.
 ```javascript
-S.find(function (x) {
-    return x === 4
-}, [0, 1, 2, 3, 4]) // => Just(4)
+> S.get('x', {x: 1, y: 2})
+Just(1)
+> S.get('toString', {x: 1, y: 2})
+Nothing()
+```
+
+<h4 name="gets"><code><a href="https://github.com/plaid/sanctuary/blob/v0.4.0/index.js#L859">gets :: [String] -> Object -> Maybe *</a></code></h4>
+
+Takes a list of property names and an object and returns Just the
+value at the path specified by the list of property names if such
+a path exists; Nothing otherwise.
 
-S.find(function (x) {
-    return x === 4
-}, [0, 1, 2, 3]) // => Nothing()
+```javascript
+> S.gets(['a', 'b', 'c'], {a: {b: {c: 42}}})
+Just(42)
+> S.gets(['a', 'b', 'c'], {})
+Nothing()
 ```
 
-Sanctuary also provide `null`-safe functions for parsing various inputs:
+### Parse
+
+<h4 name="parseDate"><code><a href="https://github.com/plaid/sanctuary/blob/v0.4.0/index.js#L879">parseDate :: String -> Maybe Date</a></code></h4>
 
-`parseDate :: String -> Maybe Date`
+Takes a string and returns Just the date represented by the string
+if it does in fact represent a date; Nothing otherwise.
 
-Returns a `Just` containing a Javascript date object if the date is valid or a `Nothing` otherwise.
 ```javascript
-S.parseDate('2011-01-19T17:40:00Z') // => Just('Sun Jun 19 2011 19:40:00 GMT+0100 (BST)')
-S.parseDate('2015-02-31T17:40:00Z') // => Nothing()
+> S.parseDate('2001-02-03')
+Just(new Date("2001-02-03T00:00:00.000Z"))
```

Comment:
Shame the date has changed there, the previous value was the exact date and time my son was born. :smile: